gradle-plugin/README.md

```diff
@@ -215,16 +215,26 @@ emerge {
     snapshotsStorageDirectory.set(
       "/src/main/snapshots"
     ) // Path to local snapshot image storage, defaults to `/build/emerge/snapshots/outputs`
+
+    defaultApiVersion.set(33) // Android API version to run snapshots on, must be within 29-34
```

Comment:
Based on the `val excludedVersions = listOf(30, 32)` code I think it would be clearer to enumerate all the supported versions instead of giving a range.

Reply:
Done!